src/data_store/table.rs

```diff
@@ -0,0 +1,105 @@
+use std::sync::Arc;
+
+use chrono::NaiveDateTime;
+use datafusion::arrow::array::{BooleanArray, Int64Array, RecordBatch, StringArray};
+use datafusion::prelude::*;
+use datafusion::arrow::datatypes::{DataType, Field, Schema};
+use serde::{Deserialize, Serialize};
+use sqlx::{types::Json, PgPool};
+
+use crate::utils::{LOAD_BALANCER_NAME, TABLE_NAME};
+use super::DataStoreError;
+
+#[derive(Serialize, Deserialize, Debug, sqlx::FromRow)]
+pub struct Table {
+    pub id: i64,
+    pub server_name: String,
+    pub worker_name: String,
+    pub port_name: String,
+    pub active: bool,
+    pub info: Option<Json<Info>>, 
+    pub inserted_at: NaiveDateTime,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Info {
+    pub cnt_conns: i32,
```

Comment:
Doubt:
Can `cnt_conns` be negative?
If not, could it be then `u32`?

Reply:
That's a good point! It can be only positive.